lib/models/Character.js

```diff
@@ -17,6 +17,13 @@ class Character {
 
   static async getAll() {
     // implement getAll() method to return all characters with a list of quotes
+    const { rows } =
+      await pool.query(`SELECT characters.*, COALESCE(json_agg(to_jsonb(quotes))
+    FILTER (WHERE quotes.id IS NOT NULL), '[]'
+    ) as quotes from characters
+    LEFT JOIN quotes on characters.id = character_id
+    GROUP BY characters.id`);
```

Comment:
To be explicit on our LEFT JOIN, let's do LEFT JOIN quotes ON character.id = quotes.character_id..